packages/fiori/src/themes/InvisibleTextStyles.css

```diff
@@ -0,0 +1,7 @@
+.ui5-hidden-text {
```

Comment:
Not for this change, but we should consider moving this somewhere else - base or theme-base

Reply:
I agree